resources/js/components/CampYearMap.vue

```diff
@@ -16,10 +16,12 @@
         v-for="x in filteredCampData"
         :key="x.id"
         :lat-lng="x.latlng"
-        :radius="x.size * 2"
-        color="red"
+        :radius="(x.num_members + x.num_participants) * 2" 
```

Comment:
Can you create a computed property for this?

Reply:
Not sure how since that's in a for loop

Reply:
I put an extra property on the CampMapData value object